django/db/models/fields/__init__.py

```diff
@@ -966,7 +966,7 @@ def get_prep_value(self, value):
         return int(value)
 
     def contribute_to_class(self, cls, name, **kwargs):
-        assert not cls._meta.auto_field, "A model can't have more than one AutoField."
+        assert not cls._meta.auto_field, "A model %s can't have more than one AutoField." % cls
```

Comment:
I think "Model %s can't..." might be better grammar. Could you add a test for the message?